fgd/brush/trigger/trigger_momentum_timer_stage.fgd

```diff
@@ -1,6 +1,11 @@
-@SolidClass base(ZoneTrigger)
-	appliesto(MOMENTUM)
-= trigger_momentum_timer_stage : "Starting trigger for each stage of a map. trigger_momentum_timer_start is automatically stage 1!" 
+@SolidClass appliesto(MOMENTUM)
+= trigger_momentum_timer_stage : "Starting zone for each stage of a map. trigger_momentum_timer_start is automatically stage 1!" 
 	[
-	teleport_destination(target_destination) : "Remote Destination" : : "The entity specifying the point to which the player should be teleported when restarting a stage."
+	targetname(target_source) : "Name" : : "The name of this zone."
```

Comment:
Is this needed?

Reply:
Stages can have names, right?

Reply:
Yeah, guess I was confused by the key being `targetname`

Reply:
```suggestion
	stage_name(string) : "Stage Name" : : "A custom name for this stage (optional)."
```